packages/design-tokens/src/scss/themes/light-inverted/_colors.scss

```
@@ -0,0 +1,275 @@
+    ),
+) !default;
+
+$custom-colors: (
+    01: (
+        default: $custom-01-default,
+        hover: $custom-01-hover,
+        active: $custom-01-active,
+        background: $custom-01-background,
+        content: $custom-01-content,
+    ),
+    02: (
+        default: $custom-02-default,
+        hover: $custom-02-hover,
+        active: $custom-02-active,
+        background: $custom-02-background,
+        content: $custom-02-content,
+    ),
+) !default;
+
+$disabled-colors: (
+    border: $disabled-border,
+    background: $disabled-background,
+    foreground: $disabled-foreground,
+    text: $disabled-text,
+) !default;
+
+$border-colors: (
+    interactive: (
+        default: $border-interactive-default,
+        hover: $border-interactive-hover,
+        active: $border-interactive-active,
+        selected: $border-interactive-selected,
+    ),
+    secondary: (
+        default: $border-secondary-default,
+    ),
+    focus: (
+        default: $border-focus-default,
+    ),
+) !default;
+
+$action-colors: (
+    button: (
+        primary: (
+            default: $action-button-primary-default,
+            hover: $action-button-primary-hover,
+            active: $action-button-primary-active,
+            content: $action-button-primary-content,
+            border: $action-button-primary-border,
+        ),
+        secondary: (
+            default: $action-button-secondary-default,
+            hover: $action-button-secondary-hover,
+            active: $action-button-secondary-active,
+            content: $action-button-secondary-content,
+            border: $action-button-secondary-border,
+        ),
+        tertiary: (
+            default: $action-button-tertiary-default,
+            hover: $action-button-tertiary-hover,
+            active: $action-button-tertiary-active,
+            content: $action-button-tertiary-content,
+            border: $action-button-tertiary-border,
+        ),
+        plain: (
+            default: $action-button-plain-default,
+            hover: $action-button-plain-hover,
+            active: $action-button-plain-active,
+            content: $action-button-plain-content,
+            border: $action-button-plain-border,
+        ),
+    ),
+    link: (
+        primary: (
+            default: $action-link-primary-default,
+            hover: $action-link-primary-hover,
+            active: $action-link-primary-active,
+        ),
+        secondary: (
+            default: $action-link-secondary-default,
+            hover: $action-link-secondary-hover,
+            active: $action-link-secondary-active,
+        ),
+        tertiary: (
+            default: $action-link-tertiary-default,
+            hover: $action-link-tertiary-hover,
+            active: $action-link-tertiary-active,
+        ),
+        visited: (
+            default: $action-link-visited-default,
+        ),
+    ),
+) !default;
+
+$background-colors: (
+    formfield: (
```

I think it's two words… 🤔 

```suggestion
    form-field: (
```